v2/util/qc_stmt.go

```diff
@@ -117,6 +119,76 @@ func checkAsn1Reencoding(i interface{}, originalEncoding []byte, appendIfCompari
 	return result
 }
 
+func GetEtsiQcTypes(c *x509.Certificate) []asn1.ObjectIdentifier {
+	var result []asn1.ObjectIdentifier
+	ext := GetExtFromCert(c, QcStateOid)
+	if ext == nil {
+		return nil
+	}
+	s := ParseQcStatem(ext.Value, IdEtsiQcsQcType)
+	if len(s.GetErrorInfo()) != 0 {
+		return nil
+	}
+	if !s.IsPresent() {
+		return result
+	}
+	qcType := s.(Etsi423QcType)
+	result = append(result, qcType.TypeOids...)
+	return result
+}
```

Comment:
Should we be ensuring these new functions use the new ZCrypto QCStatement parsing? I suspect longer-term, most of this file will be able to go away now that ZCrypto landed support in https://github.com/zmap/zcrypto/pull/217 (which was added to zlint in https://github.com/zmap/zlint/pull/435 ), so I'm trying to figure out the best way to do that.

**Option 1**: Obsolete/update qc_stmt.go in favor of the ZCrypto parsing, and then continue
**Option 2**: Incrementally adjust for this lint, and then later finish the rest of the conversion.

@cpu @dadrian Do you have thoughts here? I'm slightly inclined to **Option 2** to keep forward momentum, but I realize that leaves the file in an inconsistent state.

Based on the ZCrypto change, my understanding for Option 2 would be that this would just be
```suggestion
func GetEtsiQcTypes(c *x509.Certificate) []asn1.ObjectIdentifier {
  var result []asn1.ObjectIdentifier
  for _, qcTypeStmt := range c.QCStatements.ParsedStatements.Types {
    result = append(result, qcTypeStmt.TypeIdentifiers...)
   }
   return result
}
```

However, I'd want to hear from the maintainers before going that route. And there's probably some 'nil' handling needed for both QCStatements and ParsedStatements ;)

Reply:
I'm not sure how I missed https://github.com/zmap/zlint/issues/434 but yeah, that's what I'm flagging here.

Should that be resolved first is, I think, the question at hand :)